src/dropzone/dropzone.html

```diff
@@ -0,0 +1,12 @@
+<form submit-form
+  action="{{ $ctrl.url }}"
+  class="dropzone"
+  id="s3c-dropzone"
```

Comment:
This should probably be a `class` instead of an `id`. S3Commander is intended ot be embedded in other pages, you shouldn't assume there's only one instance per page.

Reply:
Ah, that makes sense. I'll drop this id attribute (it's not being used anywhere).

I will also go ahead and remove the uploadForm component on this branch as well.